web/src/components/base/ChipButton.tsx

```diff
@@ -1,46 +1,96 @@
-import styled from '@emotion/styled'
-import CloseIcon from '@mui/icons-material/Close'
-import { SvgIconProps } from '@mui/material/SvgIcon'
-import React, { ElementType, ReactElement } from 'react'
+import { Chip, ChipOwnProps, Theme, useTheme } from '@mui/material/SvgIcon'
```

Comment:
SvgIcon is jack of all trades 😂
<img width="400" alt="Image" src="https://github.com/user-attachments/assets/024b81c4-47a3-42ff-93ca-d28ede057072" />